src/github.rs

```diff
@@ -1374,6 +1425,7 @@ pub trait IssuesQuery {
     async fn query<'a>(
         &'a self,
         repo: &'a Repository,
+        include_fcp_details: bool,
```

Comment:
I don't love adding this param here, to be honest... feels like this repo mostly stays away from such behavior

Reply:
I think I expect this `bool` to be part of the `Self` type here

Reply:
I think it seems *ok* for now, it doesn't feel like there's an obvious right thing. We can refactor in the future as needed.